docs/src/dMCDA.md

```diff
@@ -1 +1,159 @@
-# Dynamic Multi-Criteria Decision Analysis
\ No newline at end of file
+# Dynamic Multi-Criteria Decision Analysis
+
+## Site Selection in ADRIA
+At each time step where an intervention is performed at a site, the sites for intervention can be selected either randomly or by using a site selection algorithm. When an algorithm is used, site selection is performed by iteratively updating how each site/reef in the domain performs against a range of criteria. Currently, ADRIA allows the following criteria to be used in site selection:
+
+* Connectivity, calculated as ```centrality*(site coral cover)*(site area)```.
+* Wave damage probability.
+* Heat stress probability.
+* Priority predecessors, or sites with the strongest connectivity to "priority sites" (defined by the user in the ```prioritysites``` parameter).
+* Priority zones, such as GBRMPA management zones (defined by the user in the ```priorityzones``` parameter in order of preference).
+* Coral cover relative to the site carrying capacity.
+* Site depth, for which median site depth is used.
+
+In the case of site selection for seeding, the coral cover criterium is defined as the area not occupied by coral relative to the site carrying capacity. For shading, the coral cover criterium is defined as the area already covered with coral relative to the carrying capacity. All other criteria are identical for seeding and shading, so that sites selected for these two interventions tend to be similar. The priority predecessors and priority zones criteria allow the user to incorporate spatial objectives, such as through GBRMPA management zone layers or data layers from other spatial management software such as MARXAN.
+
+These criteria are used within methods based on multi-criteria decision analysis (MCDA) to rank sites from most to least suitable for intervention at each intervention time step when ADRIA is run. The criteria are weighted according to the user's preference and the weights can be defined in ADRIA through the input scenario parameters, or inline, as in the following example:
+
+```julia
+@info "Loading data package"
+here = @__DIR__
+dom = ADRIA.load_domain(joinpath(here, "Example_domain"))
+
+@info "Creating 5 scenarios based on parameter bounds using the Sobol' method"
+scens = ADRIA.sample(dom, 5)
+
+@info "Adjusting heat_stress and wave_stress weightings"
+scens.wave_stress .= [0.7,0.6,0.1,0.4]
+scens.heat_stress .= [0.0,0.2,0.1,0.7]
+
+```
+The criteria weights and parameters which the user defines in ADRIA are described in the table below.
+
+|Parameter                    |Description                                                                        |
+|-----------------------------|-----------------------------------------------------------------------------------|
+|```heat_stress```            |Weight for heat stress when selecting seeding or shading sites.                    |
```

Comment:
Description should indicate what the weight values do.

e.g., High values has what effect? Put an emphasis on hotter locations so they are more likely to be selected?